go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/goravel/framework
 
-go 1.21
+go 1.22
```

Comment:
There are a lot of users using 1.21, so we may keep it.

Reply:
> There are a lot of users using 1.21, so we may keep it.

Can't make it, because https://github.com/sethvargo/go-limiter package's go.mod configure to go 1.22.

Reply:
I think they are compatible?

Reply:
> I think they are compatible?

But go mod tidy will auto update it to go 1.22

Reply:
> > I think they are compatible?
> 
> But `go mod tidy` will auto update it to go 1.22

OK, after full implement Store, `go-limiter` package can be remove now.